lib/core/utils/preload.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Validated the preload object
+ * @param {Object | boolean} preload configuration object or boolean passed via the options parameter to axe.run
+ * @return {boolean}
+ * @private
+ */
+function isValidPreloadObject(preload) {
+	return (
+		typeof preload === 'object' &&
+		preload.hasOwnProperty('assets') &&
```

Comment:
Can remove the `.hasOwnProperty` check, as `Array.isArray(undefined)` returns `false`.